Tests/Controller/Api/MediaControllerTest.php

```diff
@@ -31,11 +31,13 @@ public function testGetMediaAction()
 
         $mController = $this->createMediaController($mManager);
 
-        $params = $this->getMock('FOS\RestBundle\Request\ParamFetcherInterface');
-        $params->expects($this->once())->method('all')->will($this->returnValue(array('page' => 1, 'count' => 10, 'orderBy' => array('id' => 'ASC'))));
-        $params->expects($this->exactly(3))->method('get');
+        $paramFetcher = $this->getMockBuilder('FOS\RestBundle\Request\ParamFetcher')
+            ->disableOriginalConstructor()
+            ->getMock();
+        $paramFetcher->expects($this->exactly(3))->method('get');
+        $paramFetcher->expects($this->once())->method('all')->will($this->returnValue(array()));
```

Comment:
@core23 : a very similar change is missing in the Gallery controller test. It was introduced in #1155